lib/jekyll/commands/new.rb

```diff
@@ -31,10 +35,11 @@ def process(args, options = {})
           if options["blank"]
             create_blank_site new_blog_path
           else
-            create_site new_blog_path
+            create_site new_blog_title, new_blog_path
```

Comment:
Note to self: `create_site` is `private`, so changing its signature is no problem :+1: